src/server/services/image.service.ts

```diff
@@ -667,7 +667,14 @@ export const getAllImages = async ({
       orderBy = `im."tippedAmountCount" DESC, im."reactionCount" DESC, im."imageId"`;
       if (!isGallery) AND.push(Prisma.sql`im."tippedAmountCount" > 0`);
     } else if (sort === ImageSort.Random) orderBy = 'ct."randomId" DESC';
-    else orderBy = `i."id" DESC`;
+    else {
+      if (from.indexOf(`irr`)) {
```

Comment:
I believe it should be this way
```suggestion
      if (from.indexOf(`irr`) !== -1) {
```